### DISCUSSION TITLE
Ingestion issue

### DISCUSSION THREAD

**Discussion by rohandixitdelhi95** [Sunbird-cQube/community#848] [General]
After ingesting the files the data is not showing on minio console.

**Comment by RNarmatha** [comment]
Hi @rohandixitdelhi95 ,

Could you please share the error logs ?

@bommalla - Once Rohan share the details of the logs please take it guide them to fix this issue 

Thanks

**Comment by bommalla** [comment]
@rohandixitdelhi95 @RNarmatha 
Sure

**Comment by RNarmatha** [comment]
@bommalla - please post the command to check the  error logs

**Comment by bommalla** [comment]
@rohandixitdelhi95 @RNarmatha 

sudo docker logs ingest_app

**Comment by rohandixitdelhi95** [comment]
@RNarmatha @bommalla 
<img width="960" alt="image" src="https://github.com/Sunbird-cQube/community/assets/140483814/95a55b61-b44d-4a68-ba07-8a9428bcf261">

**Comment by bommalla** [comment]
@rohandixitdelhi95 @RNarmatha 
Please send me the minio logs as well

sudo docker logs cqube_minio

**Comment by rohandixitdelhi95** [comment]
@bommalla @RNarmatha 
![image](https://github.com/Sunbird-cQube/community/assets/140483814/3be5b195-856e-4993-99a7-b78667c140d9)

**Comment by bommalla** [comment]
@rohandixitdelhi95 @RNarmatha 

Please check the below command send me the screenshot

sudo docker ps -a

**Comment by rohandixitdelhi95** [comment]
![image](https://github.com/Sunbird-cQube/community/assets/140483814/83d97860-cdf5-496e-a33f-0e70d5d7e0a7)

**Comment by bommalla** [comment]
@rohandixitdelhi95 @RNarmatha 

Please check the above screenshot the ingest_app container is not running 

Please check and rebuild the container again.

**Comment by RNarmatha** [comment]
Hi @rohandixitdelhi95 -

could you please update the status ?

**Comment by RNarmatha** [comment]
@bommalla - please provide the commands to rebuild the containers

**Comment by bommalla** [comment]
@rohandixitdelhi95 @RNarmatha 

Any update on this issue ?

**Comment by bommalla** [comment]
@rohandixitdelhi95 @RNarmatha 

I scheduled call at 10:30 today please be available at the time.
